### PR TITLE
Rights statement button

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: components/generic/RightsStatementButton.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  import RightsStatement from '../../components/generic/RightsStatement';
+  import SmartLink from '../../components/generic/SmartLink';
+
+  export default {
+    components: {
+      RightsStatement,
+      SmartLink
+    },
+    props: {
+      rightsStatementIsUrl: {
```
Comment (rwd) [inline components/generic/RightsStatementButton.vue:24]: I don't think this really makes sense as a prop as it is conditional for rendering the entire template. This should instead be left to the parent view to determine whether or not to even render the RightsStatementButton component. Remove this prop.

File: components/generic/RightsStatementButton.stories.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const i18n = new VueI18n();
+
+storiesOf('Generic', module)
+  .add('Rights Statement Button', () => ({
+    components: { RightsStatementButton },
+    i18n,
+    data() {
+      return {
+        rightsStatementIsUrl: true,
+        rightsStatement: 'https://creativecommons.org/licenses/by-sa/4.0/',
```
Comment (rwd) [inline components/generic/RightsStatementButton.stories.js:14]: Remove the trailing comma.

File: components/generic/RightsStatementButton.stories.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      };
+    },
+    template: `
+      <b-container class="mt-3 bg-white p-5">
+        <RightsStatementButton
+          v-if="rightsStatementIsUrl"
+          :rights-statement="rightsStatement"
+        />
+      </b-container>
+    `
```
Comment (rwd) [inline components/generic/RightsStatementButton.stories.js:24]: Remove the blank line.

File: components/generic/RightsStatementButton.vue [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <SmartLink
```
Comment (rwd) [inline components/generic/RightsStatementButton.vue:2]: This results in the icon for external links showing, but in the design, and as clarified during sprint refinement, in this context it is not wanted. Could you come up with a way to not show it here, but to continue showing it in all other contexts, likely via a prop sent down to SmartLink.

File: components/generic/RightsStatementButton.vue (hunk 1 above)
Comment (alansutherland) [reply components/generic/RightsStatementButton.vue:24]: true, I have removed it

File: components/generic/RightsStatementButton.stories.js (hunk 2 above)
Comment (alansutherland) [reply components/generic/RightsStatementButton.stories.js:14]: removed

File: components/generic/RightsStatementButton.stories.js (hunk 3 above)
Comment (alansutherland) [reply components/generic/RightsStatementButton.stories.js:24]: removed

File: components/generic/RightsStatementButton.vue (hunk 4 above)
Comment (alansutherland) [reply components/generic/RightsStatementButton.vue:2]: I have added the hideExternalIcon prop, other smartLinks will still show the external link icon

**Pull request merged**